multi-node/aws/README.md

```diff
@@ -6,14 +6,30 @@ View the full instructions at https://coreos.com/kubernetes/docs/latest/kubernet
 
 ### Download pre-built binary
 
+First, make sure you have imported the [CoreOS Application Signing Public Key](https://coreos.com/security/app-signing-key/).
+
+```sh
+gpg2 --keyserver pgp.mit.edu --recv-key FC8A365E
+
+#Validate the key fingerprint
+gpg2 --fingerprint FC8A365E
+# Key fingerprint = 18AD 5014 C99E F7E3 BA5F  6CE9 50BD D3E0 FC8A 365E
```

Comment:
The actual output is depends on their trust state and also lists subkeys (which will grow over time), so we've just been showing the primary key fingerprint in etcd release notes and coreos-baremetal, then linking to the full key.
    
    gpg key fingerprint is: 18AD 5014 C99E F7E3 BA5F  6CE9 50BD D3E0 FC8A 365E